website/pages/api-docs/acl-policies.mdx

```diff
@@ -20,8 +20,8 @@ to the region, and may lag behind the authoritative region.
 | `GET`  | `/acl/policies` | `application/json` |
 
 The table below shows this endpoint's support for
-[blocking queries](/api#blocking-queries), [consistency modes](/api#consistency-modes) and
-[required ACLs](/api#acls).
+[blocking queries](/api-docs#blocking-queries), [consistency modes](/api-docs#consistency-modes) and
```

Comment:
The `/api` -> `/api-docs` change makes me sad, but if our tools force that on us that's fine. What's another mountain of redirects? :grimacing: